src/svg.draw.js

```diff
@@ -1,248 +1,242 @@
-    // Our Object which manages drawing
-    function PaintHandler(el, event, options) {
-
-        this.el = el;
-        el.remember('_paintHandler', this);
-    
-        var _this = this,
-            plugin = this.getPlugin();
-
-        this.parent = el.parent(SVG.Nested) || el.parent(SVG.Doc);
-        this.p = this.parent.node.createSVGPoint(); // Helping point for coord transformation
-        this.m = null;  // transformation matrix. We get it when drawing starts
-        this.startPoint = null;
-        this.lastUpdateCall = null;
-        this.options = {};
-        this.set = new SVG.Set();
-
-        // Merge options and defaults
-        for (var i in this.el.draw.defaults) {
-            this.options[i] = this.el.draw.defaults[i];
-            if (typeof options[i] !== 'undefined') {
-                this.options[i] = options[i];
-            }
-        }
-        
-        if(plugin.point) {
-          plugin['pointPlugin'] = plugin.point;
-          delete plugin.point;
-        }
-        
-        // Import all methods from plugin into object
-        for (var i in plugin){
-            this[i] = plugin[i];
-        }
-        
-        // When we got an event, we use this for start, otherwise we use the click-event as default
-        if (!event) {
-            this.parent.on('click.draw', function (e) {
-                _this.start(e);
-            });
-
-        }
+import { Element, Nested, Doc, extend, on, off } from '@svgdotjs/svg.js'
 
+const defaults = { snapToGrid: 1 }
+let drawPlugins = {}
+
+// Our Object which manages drawing
+class PaintHandler {
+
+  constructor (el, event, options) {
+
+    this.el = el
+    el.remember('_paintHandler', this)
+
+    this.start = this.start.bind(this)
+    this.point = this.point.bind(this)
+    this.stop = this.stop.bind(this)
+    this.update = this.update.bind(this)
+    this.done = this.done.bind(this)
+    this.cancel = this.cancel.bind(this)
+
+    var plugin = this.getPlugin()
+
+    this.parent = el.parent(Nested) || el.parent(Doc)
```

Comment:
`Nested` doesnt exist anymore. It's all `Doc` now. Just use `el.parent()` without any parameter